Restructure "Module" menu in header navigation

oliverguenther opened the pull request:
This effectively changes the following menus:

**Top Menu**
- Removed Administration item
- Added Item 'Work Packages' -> /work_packages
- Added Item 'News' -> /news
- Added Item 'Time Sheet' -> /time_entries (if user has a project with
      view time log permission)

**My Menu**
- Added Administration item as first position

Relevant work package:
https://community.openproject.org/work_packages/20269

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://oliverguenther-feature-consolidate-module-menu.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://oliverguenther-feature-consolidate-module-menu.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

oliverguenther committed:
Consolidate Module menu in header navigation
This effectively changes the following menus:

**Top Menu**

- Removed Administration item
- Added Item 'Work Packages' -> /work_packages
- Added Item 'News' -> /news
- Added Item 'Activity' -> /activity
- Added Item 'Time Sheet' -> /time_entries (if user has a project with
        view time log permission)

**My Menu**

- Added Administration item as first position

oliverguenther committed:
Remove superfluous buttons in projects overview

With work packages, time entries, news, and activity now
residing in the top menu, these buttons are superfluous
in the projects overview.

Relevant work package:
https://community.openproject.org/work_packages/20512

oliverguenther committed:
Remove activity item for 4.2

floriank commented:
@oliverguenther: does this by any chance partially duplicate https://community.openproject.org/work_packages/18364?

oliverguenther commented:
Thanks @floriank , must have overseen this one when I consolidated the relevant WPs with Niels.

If I'm not forgetting something, then [#20269](https://community.openproject.org/work_packages/20269) and [#20503](https://community.openproject.org/work_packages/20503) implements your #18364.

floriank commented:
Great! Less work for me. I'll go ahead and close my WP.

oliverguenther commented:
Rebased against release/4.2 in https://github.com/opf/openproject/pull/3159
